client/my-sites/checkout/composite-checkout/test/composite-checkout-thank-you.js

```diff
@@ -635,18 +635,16 @@ describe( 'getThankYouPageUrl', () => {
 		expect( url ).toBe( '/cookie/:receiptId' );
 	} );
 
-	// Note: This just verifies the existing behavior; I suspect this is a bug
-	it( 'redirects to thank-you page followed by placeholder receiptId twice if no cookie url is set, create_new_blog is set, and there is no receipt', () => {
+	it( 'redirects to thank-you page followed by placeholder receiptId if no cookie url is set, create_new_blog is set, and there is no receipt', () => {
```

Comment:
Oh thank the gods; this always bothered me ever since I migrated this code into its current location (hence the comments).